### ISSUE TITLE
Importing .ics files is broken

### ISSUE THREAD

**Issue by teonnik** [pimutils/khal#1163]
After the most recent update to `khal` I get the following error when I try to import an `.ics`  file:

```
warning: /dev/tty does not exist, importing might not work
.... <event info> ....
Do you want to import this event into `private`? [y/N]: Aborted!
Press any key to continue...
```

and the event is not imported.

----

khal version: 0.10.5

System:

```
OS: Arch Linux x86_64 
Host: 20L7CTO1WW ThinkPad T480s 
Kernel: 5.18.9-arch1-1 
Shell: zsh 5.9 
WM: sway 
Terminal: foot 
CPU: Intel i7-8550U (8) @ 4.000GHz 
GPU: Intel UHD Graphics 620 
```

**Comment by Role92** [comment]
You can put this in your `mailcap` file:
```
text/calendar; khal import %s;
```

With this line, `.ics` files can just be opened from the attachment view, since they have the unique mimetype `text/calendar`.

**Comment by teonnik** [comment]
I am aware that this is an option but I have `text/calendar; vcal %s; copiousoutput` in my mailcap already... this allows me to view the event within neomutt. That is why I have the macro defined above, so that I can do the equivalent with `Ctrl+k`.

**Comment by geier** [comment]
@teonnik thank you for reporting that bug.

Can you confirm it worked with khal 0.10.4?

**Comment by bbbart** [comment]
I cannot confirm if it worked before, but I am getting the exact same error message, either when trying to import an .ics through a mutt macro, or directly from the command line.

Using 0.10.5

**Comment by Djedouas** [comment]
Same problem here.

Import is broken when using pipe, from the command line or from aerc mail client.

**Comment by Djedouas** [comment]
Yes it is failing because of the confirmation, so --batch works because there is no confirmation.

But it seems that it is supposed to work with confirmation and pipe feature... which is what I would like to have
